diesel/src/expression/mod.rs

```diff
@@ -165,6 +165,47 @@ use query_builder::{QueryFragment, QueryId};
 /// Helper trait used when boxing expressions. This exists to work around the
 /// fact that Rust will not let us use non-core types as bounds on a trait
 /// object (you could not return `Box<Expression+NonAggregate>`)
+///
+/// # Examples:
```

Comment:
No. Those are wrong. There are no colons after headlines when the headline actually printed as a headline. That's just as bad as people who put a space before an exclamation mark ! It's not even correct to do that in French because there you use a _thin space_ or even a _hair space_  ! Typography is art!!11one `</rant>` 😆